src/fetch/transfer_file.py

```diff
@@ -108,6 +110,16 @@ def from_dict(cls, obj: dict[str, str]) -> Transfer:
             amount_wei=int(obj["amount"]),
         )
 
+    @classmethod
+    def from_dataframe(cls, pdf: pd.DataFrame) -> Transfer:
+        """Converts Dune data dict to object with types"""
+        token_address = pdf.get("token_address", None)
```

Comment:
I also assume we only get back 1 row here. So it might be meaningful to change the method name from `from_dataframe` to `from_datarow` or something similar. What do you think?

Reply:
This method isn't actually correct yet. I'm not even sure if we will actually use a data frame. Sorry to waste your time here.